Automattic/wp-calypso
# Revert "Gutenberg blocks: use SVG primitives to add accessibility attributes"

Reverts Automattic/wp-calypso#28752

Caused `Uncaught TypeError: Cannot read property 'SVG' of undefined` in Map block's view script.

We'll probably need to add `wp-components` as a dependency for Map block's view script; I'll have another look tomorrow.

## Testing instructions
- Launch Gutenberg gutenpack-jn
- Insert map block and save the post/page
- Confirm no `Uncaught TypeError: Cannot read property 'SVG' of undefined` at frontend
